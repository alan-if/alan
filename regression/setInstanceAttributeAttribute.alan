Every roomWithDoors Isa location
  Has doors 2.
End Every roomWithDoors.

The l Isa roomWithDoors
  Has doors 4.
End The l.

The k Isa object
  Has room l.
End The k.

Start At l.
  Say doors Of l.
  Set doors Of room Of k To 5.
  Say doors Of l.
