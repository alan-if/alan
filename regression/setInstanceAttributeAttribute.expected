########## setInstanceAttributeAttribute ##########

        No warnings or errors detected.


4 5


L

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
